modules/apps/digital-signature/digital-signature-impl/src/main/java/com/liferay/digital/signature/internal/configuration/DigitalSignatureConfiguration.java

```diff
@@ -27,13 +27,14 @@
 )
 @Meta.OCD(
 	id = "com.liferay.digital.signature.internal.configuration.DigitalSignatureConfiguration",
-	localization = "content/Language", name = "digital-signature-configuration"
+	localization = "content/Language",
+	name = "digital-signature-configuration-name"
 )
 public interface DigitalSignatureConfiguration {
 
 	public boolean enable();
 
-	public String apiUsername();
+	public String apiUserName();
```

Comment:
Hi @ling-alan-huang.
Isn't necessary.
Was added suppress CamelCaseNameCheck in modules/apps/digital-signature/source-formatter-suppressions.xml and merged today.
We need to use apiUserName instead apiUsername.